src/app/shared/example-viewer/example-viewer.ts

```diff
@@ -44,4 +50,12 @@ export class ExampleViewer {
   exampleFileUrl(extension: string) {
     return `/assets/examples/${this.example}-example-${extension.toLowerCase()}.html`;
   }
+
+  copySource(text: string) {
+    if (this.copier.copyText(text)) {
+      this.snackbar.open('Code Copied', '', {duration: 800});
```

Comment:
I would make this `'Code copied'`, lowercase second `c`.

Reply:
Agreed!